### PR TITLE
fix(docs): Auto-detected typos

### PR DESCRIPTION
**Pull request by jesse-c**
These were detected with typos [1].

[1] https://github.com/crate-ci/typos

### CODE REVIEW

File: docs/source/contents/architecture/dataflow.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Initial release of Seldon Core introduced a concept of an _inference graph_, whi
 
 ![inference_graph](inference_graph.png)
 
-In reality though this was not how Seldon Core v1 is implemented. Instead, Seldon deployment consists of a range of independent services that host models, transofrmations, detectors and explainers, and a central orchestrator that knows the inference graph topology and makes service calls in the correct order, passing data between response and requests as necessary. Here is how the picture looks under the hood:
+In reality though this was not how Seldon Core v1 is implemented. Instead, Seldon deployment consists of a range of independent services that host models, transformations, detectors and explainers, and a central orchestrator that knows the inference graph topology and makes service calls in the correct order, passing data between response and requests as necessary. Here is how the picture looks under the hood:
```
Comment (KengoA) [inline docs/source/contents/architecture/dataflow.md:11]: very minor nit:
```suggestion
In reality though this was not how Seldon Core v1 is implemented. Instead, Seldon deployment consists of a range of independent services that host models, transformations, detectors and explainers, and a central orchestrator that knows the inference graph topology and makes service calls in the correct order, passing data between requests and responses as necessary. Here is how the picture looks under the hood:
```

File: docs/source/contents/drift/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ An example drift detection model from the CIFAR10 image classification example i
 :language: yaml
 ```
 
-Usually you would run these model in an asynchronous part of a Pipeline, i.e. they are not connected to the output of the Pipeline which defines the sychronous path. For example the CIFAR10 image detection example uses a pipeline as shown below:
+Usually you would run these model in an asynchronous part of a Pipeline, i.e. they are not connected to the output of the Pipeline which defines the synchronous path. For example the CIFAR10 image detection example uses a pipeline as shown below:
```
Comment (KengoA) [inline docs/source/contents/drift/index.md:11]:
```suggestion
Usually you would run these model in an asynchronous part of a Pipeline, i.e. they are not connected to the output of the Pipeline which defines the synchronous path. For example the CIFAR-10 image detection example uses a pipeline as shown below:
```

File: docs/source/contents/getting-started/kubernetes-installation/security/index.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Seldon can be run with secure control plane and data plane operations. There are
  * [Kafka](#kafka)
  * [Data Plane](#data-plane)
 
-The various communication points betwwen services is shown in the diagram below:
+The various communication points between services is shown in the diagram below:
```
Comment (KengoA) [inline docs/source/contents/getting-started/kubernetes-installation/security/index.md:9]:
```suggestion
The various communication points between services are shown in the diagram below:
```

File: docs/source/contents/kubernetes/resources/seldonconfig/index.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 This section is for advanced usage where you want to define how seldon is installed in each namespace.
 ```
 
-The SeldonConfig resource defines the core installation components installed by Seldon. If you wish to install Seldon you can use the [SeldonRuntime](../seldonruntime/index.md) resource which allow easy overriding of some parts defined in this specificiation. In general, we advise core DevOps to use the default SeldonConfig or customize it for their usage. Individual installation of Seldon can then use the SeldonRuntime with a few overrides for special customization needed in that namespace.
+The SeldonConfig resource defines the core installation components installed by Seldon. If you wish to install Seldon you can use the [SeldonRuntime](../seldonruntime/index.md) resource which allow easy overriding of some parts defined in this specification. In general, we advise core DevOps to use the default SeldonConfig or customize it for their usage. Individual installation of Seldon can then use the SeldonRuntime with a few overrides for special customization needed in that namespace.
```
Comment (KengoA) [inline docs/source/contents/kubernetes/resources/seldonconfig/index.md:7]:
```suggestion
The SeldonConfig resource defines the core installation components installed by Seldon. If you wish to install Seldon, you can use the [SeldonRuntime](../seldonruntime/index.md) resource which allows easy overriding of some parts defined in this specification. In general, we advise core DevOps to use the default SeldonConfig or customize it for their usage. Individual installation of Seldon can then use the SeldonRuntime with a few overrides for special customisation needed in that namespace.
```

**Review by KengoA — APPROVED** [review]
Small nits and grammar fixes but overall LGTM

**Review by agrski — APPROVED** [review]
LGTM.

Presumably the exact invocation was `typos -w docs/`?  I've just installed with `asdf` and have the same set of files changed if I use that invocation.

**Comment by jesse-c** [comment]
@KengoA: Great additions, thank you!

**Pull request merged**